docs/UserGuide.md

```diff
@@ -1,12 +1,16 @@
----
-layout: page
-title: User Guide
----
+<frontmatter>
+  layout: default.md
+  title: "User Guide"
+  pageNav: 3
+  pageNavTitle: "Table Of Content"
+</frontmatter>
```

Comment:
This too can be a `<box>`:
![image](https://user-images.githubusercontent.com/1673303/218648400-c62f5442-cf88-4cae-9a43-27bf40d8b08b.png)